common.mk

```diff
@@ -48,11 +48,11 @@ help: ## Show this help.
 
 .PHONY: setup
 setup: install-uv setup-precommit ## Set up a development environment
-	uv sync $(UV_TEST_GROUPS) $(UV_LINT_GROUPS) $(UV_DOCS_GROUPS)
+	uv sync $(UV_TEST_GROUPS) $(UV_LINT_GROUPS) $(UV_DOCS_GROUPS) --reinstall-package craft-application
```

Comment:
what's this for?

Reply:
Needed while we're using craft-application from a branch (so it reinstalls with the latest version from that branch)

Reply:
Exactly. I will remove it before merging.